pkg/controllers/cache.go

```diff
@@ -108,8 +108,10 @@ type eventToChannelHandler struct {
 	name       string
 }
 
-func (e *eventToChannelHandler) OnAdd(obj interface{}) {
-	e.queueEventForObject(nil, obj)
+func (e *eventToChannelHandler) OnAdd(obj interface{}, isInInitialList bool) {
+	if !isInInitialList {
```

Comment:
Where was this change inspired from? I'm wondering if this could issue for certain operators that expect to always reconcile at first

Reply:
The `ResourceEventHandler` interface got changed, but as you are mentioning, using that `!isInInitialList` logic doesn't make sense here.